packages/frontend/app/lti/(embed)/(pages)/page.tsx

```diff
@@ -0,0 +1,167 @@
+      })
+      .catch((error) => {
+        console.error(error)
+        message.error(
+          'Failed to fetch semesters for organization with id: ' +
+            userInfo.organization?.orgId,
+        )
+      })
+  }, [userInfo.organization?.orgId])
+
+  if (shouldClose) {
+    return (
+      <main
+        className={'container mx-auto h-auto w-full max-w-lg pt-10 text-center'}
+      >
+        <title>HelpMe | Reload page</title>
+        <div className="container mx-auto h-auto w-full pt-10 text-center">
+          <Alert
+            message="End Session"
+            description="This tab/window should close automatically. Please close this tab/window and return to the LTI launch on the platform."
+            type="info"
+          />
+        </div>
+      </main>
+    )
+  }
+
+  return (
+    <>
+      <div className="mt-4 flex items-center rounded-[10px] bg-white p-3 shadow md:text-left lg:p-5 xl:max-w-[1500px]">
+        <Image
+          src={`/api/v1/organization/${userInfo?.organization?.orgId}/get_logo/${userInfo?.organization?.organizationLogoUrl}`}
+          className="mr-2 max-h-20 object-contain object-center p-1 md:max-h-80"
+          alt="Organization Logo"
+          width={80}
+          height={80}
+        />
+        <div>
+          <h1>{userInfo?.organization?.organizationName}</h1>
+          <p className="my-0">
+            {userInfo?.organization?.organizationDescription}
+          </p>
+        </div>
+      </div>
+      <div className="mt-5 flex items-center justify-between align-middle">
+        <h1 className="mt-0">My Courses</h1>
+      </div>
+      <div className="flex min-h-96 items-start justify-center">
+        {userInfo?.courses?.filter((userCourse) => userCourse.course.enabled)
+          .length === 0 ? (
+          <Empty
+            className="max-h-min"
+            description={
+              <div className={'flex w-full justify-center'}>
+                <div className={'flex w-1/4 flex-col gap-2'}>
+                  <p className={'font-semibold'}>
+                    You are not enrolled in any courses.
+                  </p>
+                  {lmsInfo && lmsInfo.platform && (
+                    <>
+                      <p>
+                        This page is being displayed in {lmsInfo.platform}. The
+                        course in {lmsInfo.platform} it is being displayed for
+                        may be linked with a HelpMe equivalent. Try refreshing
+                        the page.
+                      </p>
+                      <p>
+                        If refreshing the page does not work, contact your
+                        instructor to ask them to connect their HelpMe course
+                        with this {lmsInfo.platform} course so you can be
+                        automatically added to their course.
+                      </p>
+                      <p>
+                        If you&#39;re logged into an account which does not
```

Comment:
```suggestion
                        If you&#39;ve logged into HelpMe with a different email
```
New github PR editor has a nasty bug where I making suggestions across multiple lines switches files, so this is getting split across two suggestions